CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 ### Changed
+* Make select error description more informative in case
+  of disabled/uninit storage (#229).
```

Comment:
AFAIU, we should add 'when merger built-in module or tuple-merger external module is used'. The 'select_old' case works correctly even before the patch.

Reply:
Added.